boards/arm/atsamr21_xpro/atsamr21_xpro.dts

```diff
@@ -82,10 +82,10 @@
 		reg = <0x0>;
 		label = "RF2XX_0";
 		spi-max-frequency = <8000000>;
-		irq-gpios = <&portb 0 0>;
-		reset-gpios = <&portb 15 0>;
-		slptr-gpios = <&porta 20 0>;
-		dig2-gpios = <&portb 17 0>;
+		irq-gpios = <&portb 0 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>;
```

Comment:
The RF2XX data sheet doesn't mention anything about not driving output pins, like IRQ, in sleep state. It seems like there is no need to configure pin pull downs. Nevertheless, I preserved the existing configuration since I don't have a hardware to test.

Reply:
Yes, is not clear for sure. The section **1.3 Digital Pins** we got about DIGx pins. For IRQ pin, since entering on the TRX_OFF state from P_ON, SLEEP, DEEP_SLEEP or RESET state is indicated by interrupt IRQ_4 (AWAKE_END), if enabled, we can assume this pin have some level at sleep. The major problem with IRQ is user configurable and can be level High or level Low. Today I'm setting the default (level High). The MCLK is low if disabled.

The most important are RESET and SLPTR. This pins MUST NOT float any time otherwise radio will be at unstable state.

Today radio don't have any Power Manage functions implemented and all this can be revised in future. For the moment, I would keep same values.